angularSTEP/src/app/upload-recipe/upload-recipe.component.ts

```diff
@@ -76,44 +108,55 @@ export class UploadRecipeComponent {
     return formattedName;
   }
 
-  removeNumsAndSymbolsAtStart(originalString: string): string | undefined {
-      var startIndex: number = -1;
+  removeNumsAndSymbolsAtStart(originalString: string): string {
+    var startIndex: number = -1;
 
-      for (let i = 0; i < originalString.length; i++) {
-        var currentValue: number = originalString.charCodeAt(i);
-        if ((65 <= currentValue && currentValue <= 90) || (97 <= currentValue && currentValue <= 122)) {
-          return originalString.substring(i);
-        }
-      }
-      if (startIndex == -1) {
-        return '';
+    for (let i = 0; i < originalString.length; i++) {
+      var currentValue: number = originalString.charCodeAt(i);
+      if ((65 <= currentValue && currentValue <= 90) || (97 <= currentValue && currentValue <= 122)) {
+        return originalString.substring(i);
       }
+    }
+    if (startIndex == -1) {
       return '';
+    }
+    return '';
+  }
+
+  formatArrays(originalArray: FormArray) {
+    for (let i = 0; i < originalArray.length; i++) {
+      originalArray.at(i).setValue(this.autoCapitalizeFirst(originalArray.at(i).value));
+    }
+  }
+
+  formatInstructions(originalArray: FormArray) {
+    for (let i = 0; i < originalArray.length; i++) {
+      originalArray.at(i).setValue(this.autoCapitalizeFirst(this.removeNumsAndSymbolsAtStart(originalArray.at(i).value)));
+    }
   }
 
+  ngOnInit(): void {
```

Comment:
Does this empty function even need to be defined?

Reply:
It does not. I'll remove it.